docs/user/tipsandtricks.md

```diff
@@ -163,6 +163,14 @@ For context, installing the TensorFlow 2.7.0 CUDA-enabled variant, `tensorflow=2
 
 ## Using PyPy as an interpreter
 
+:::warning
+
+As of August 2024, [PyPy support in conda-forge has been sunset](/news/2024/08/14/sunsetting-pypy/).
+It still may be possible to create an environment using an old version of PyPy, but you will not be
+getting updates as new package versions are released.
```

Comment:
The formulation here is a bit misleading IMO. It's worse than "not getting updates". It's being stuck in ~2023. Also, pypy should still be usable with noarch packages, at least those until we bumped `python_min` to 3.10 this October.
```suggestion
It still may be possible to create an environment using an old version of PyPy, but (with the
exception of noarch packages still compatible with python 3.9), you will be constrained to the
packages that were built for PyPy until mid-2024.
```